primitives/trie/src/lib.rs

```diff
@@ -211,15 +211,13 @@ pub fn read_trie_value_with<
 
 /// Determine the default child trie root.
```

Comment:
default here confuse with default child trie type, maybe the doc should specify that this is for empty child tries, or this functions could be renamed empty_child_trie_root